desktop/src/com/mygdx/game/client/network/DesktopMaterialHandler.java

```diff
@@ -38,10 +31,7 @@ public DesktopMaterialHandler(
 
   @Override
   public boolean handle(WebSocket webSocket, int worldEntity, PlayerMaterial component) {
-    log.info("Read material component " + worldEntity);
-    var material = playerplayerMaterialMapper.create(worldEntity);
-    material.setMaterial(component.getMaterial());
-    material.setValue(component.getValue());
+    log.info("Desktop read material component " + worldEntity);
     worldHUD.prepareHudSceleton();
```

Comment:
właśnie wolę żeby to było w jednym handlerze, bo jak ten corowy wykona się później niż ten to nie będzie widać surowców przyrostu

Reply:
zauważyłem gdzieś że jeden handler wykonywał się wcześniej i to mi przeszkadzało

Reply:
a może tak się zdarzyć?

Reply:
to dziwne, bo one się już powinny teraz wykonywać sekwencyjnie :/

Reply:
czyli powinna zależeć kolejność ich umieszczenia w ComponentMessegeListenerModule?

Reply:
w tym tak

problemem są te miejsca gdzie jest @BindsInto czy coś, bo to nie zapewnia kolejności